cogeo_mosaic/mosaic.py

```diff
@@ -297,7 +301,12 @@ def from_urls(
             raise MultipleDataTypeError("Dataset should have the same data type")
 
         return cls._create_mosaic(
-            features, minzoom=minzoom, maxzoom=maxzoom, quiet=quiet, **kwargs
+            features,
+            minzoom=minzoom,
+            maxzoom=maxzoom,
+            tilematrixset=tilematrixset,
```

Comment:
if None, tms will default to `WebMercatorQuad` directly in the `_create_mosaic` method